pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -232,16 +241,29 @@ def fetch_remote_url(remote):
     def normalize_url(url):
         return url.rstrip("/") + "/"
 
-    remote_url = normalize_url(remote.url)
-    if get_repomd_file(remote, remote_url):
-        return remote_url
+    try:
```

Comment:
Similarly, get_repomd_file() will either return not-None or raise on exception. maybe this could be a try/except/else construct?